src/payment/strategies/googlepay/googlepay-adyenv2-payment-processor.ts

```diff
@@ -0,0 +1,102 @@
+import { getScriptLoader, getStylesheetLoader } from '@bigcommerce/script-loader';
+import { some } from 'lodash';
+
+import { Payment, PaymentActionCreator, PaymentInitializeOptions } from '../..';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { MissingDataError, MissingDataErrorType, NotInitializedError, NotInitializedErrorType, RequestError } from '../../../common/error/errors';
+import { PaymentMethodCancelledError } from '../../errors';
+import { AdyenAction, AdyenAdditionalAction, AdyenAdditionalActionState, AdyenClient, AdyenError, AdyenV2ScriptLoader } from '../adyenv2';
+
+export default class GooglePayAdyenV2PaymentProcessor {
+    private _adyenClient?: AdyenClient;
+    private _scriptLoader: AdyenV2ScriptLoader;
+
+    constructor(private _store: CheckoutStore, private _paymentActionCreator: PaymentActionCreator) {
+        const scriptLoader = getScriptLoader();
+        this._scriptLoader = new AdyenV2ScriptLoader(scriptLoader, getStylesheetLoader());
```

Comment:
any reason why this is not provided via the constructor, instead of instantiating it here? That way you will be able to properly mock and unit tests calls to scriptLoader